src/components/VenueBody/BookDrawer.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import {
+  StyledBookDrawerWrapper,
+  ToggleButton,
+  ButtonWrapper,
+  StyledDatePickerContainer,
+  StyledDatePickerBodyWrapper,
+} from "./Venue.styled.js";
+import DatePicker from "./DatePicker/index.jsx";
+
+export const BookDrawer = ({ venueDetails }) => {
+  const [drawerOpen, setDrawerOpen] = useState(false);
```

Comment:
The variable name is bad. It sounds like it is a function. Variable names of boolean type should start with `is` or similar prefix.

Reply:
done